tests/restv1/PayPal/TicketArchiveByPostCest.php

```diff
@@ -171,10 +171,10 @@ public function should_allow_getting_paginated_results(Restv1Tester $I) {
 	}
 
 	protected function invalid_include_post() { return [
-			'empty_string'      => [ '' ],
-			'non_existing'      => [ '23' ],
-			'bad_list_1'        => [ 'foo, bar' ],
-			'non_existing_list' => [ '23, 89' ],
-		];
+		'empty_string'      => [ '' ],
+		'non_existing'      => [ '1234124312323' ],
+		'bad_list_1'        => [ 'foo, bar' ],
+		'non_existing_list' => [ '21231233, 831231239' ],
+	];
 	}
```

Comment:
I know it was like this before, but I feel like the `return [` line should be on the next line, then all of the lines within the `invalid_include_post` function/method be indented once.